### ISSUE TITLE
Missing mounted routes when having catch-all defined

### ISSUE THREAD

**Issue by morgoth** [basecamp/google_sign_in#17]
Having google_sign_in included in Rails app and router:
```ruby
Rails.application.routes.draw do
  match "*catch_unmatched_route", via: :all, to: "application#not_found"
end
```
mounted routes from engine are not visible in the app as "catch all" route is defined "before":
```
 bin/rails routes
                   Prefix Verb URI Pattern                                                                              Controller#Action
                               /*catch_unmatched_route(.:format)                                                        application#not_found

           google_sign_in      /google_sign_in                                                                          GoogleSignIn::Engine

Routes for GoogleSignIn::Engine:
authorization POST /authorization(.:format) google_sign_in/authorizations#create
     callback GET  /callback(.:format)      google_sign_in/callbacks#show
```

I'm wondering if this somehow can be fixed in gem?
I noticed that the same problem is for active storage routes (or actually any coming from Rails).

Mounting routes by hand like:
```ruby
mount GoogleSignIn::Engine, at: Rails.configuration.google_sign_in.root || "google_sign_in", as: :fix_google_sign_in
```
workarounds the problem.

**Comment by georgeclaghorn** [comment]
I think 52f71b3a fixes this. Can you bundle `google_sign_in` from master and try it out?

**Comment by morgoth** [comment]
yes, it fixes it - I didn't notice the commit.
Thank you.

**Comment by georgeclaghorn** [comment]
Thanks for confirming! I just published v1.1.1 with the fix.
